### ISSUE TITLE
Complete polls show as 'poll'

### ISSUE THREAD

**Issue by maiki** [magicalraccoon/tootstream#205]
```
Tootstream version: 0.3.6
You are connected to https://mastodon.sdf.org
Enter a command. Use 'help' for a list of commands.


[@maiki (default)]: note
'poll'
[@maiki (default)]: 
```

I believe polls are a hot new feature, and I'm not sure what would be appropriate here. I thought it was an error, but that was a poll I had voted on and ended, hence the message to me.

Let me know if I can help test anything. ^_^

**Comment by craigmaloney** [comment]
I think we'll need to get that into https://github.com/halcy/Mastodon.py before we can incorporate that in to tootstream.

**Comment by craigmaloney** [comment]
I have a rudimentary fix for this in 0.3.7rc1,
